eng/pipelines/ci.yml

```diff
@@ -2,16 +2,16 @@
 # This file is generated automatically and any changes will be lost.
```

Comment:
didn't we want to call this file client.yml, at least for now?

Reply:
Fixed